src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,38 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: $card-gap;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    @include card($card-height, $card-width);
+    border: 1px solid $color-border;
+    border-radius: 5px;
+    color: $color-primary;
+    transition: background-color $duration, color $duration;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $color-primary;
+      color: $color-secondary;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    @include grid-col(4);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    @include grid-col(3);
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    @include grid-col(2);
+  }
+
+  @media (max-width: 487px) {
+    @include grid-col(1);
+  }
```

Comment:
you can use auto-fill instead of all these media, but media also good solution

Reply:
> you can use auto-fill instead of all these media, but media also good solution

Thank you, I've tried "auto-fill". Works silmilar, but looks even cleaner.